pkgs/applications/editors/vscode/extensions/contextmapper.context-mapper-vscode-extension/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, vscode-utils, substituteAll, graphviz, jdk11, makeWrapper }:
+vscode-utils.buildVscodeMarketplaceExtension rec {
```

```suggestion
{ lib, vscode-utils, substituteAll, graphviz, jdk11, makeWrapper }:

vscode-utils.buildVscodeMarketplaceExtension rec {
```
It is usual to skip a line after arguments.